src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/AmiNameResolverIT.kt

```diff
@@ -0,0 +1,91 @@
+package com.atlassian.performance.tools.awsinfrastructure
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.regions.Regions.*
+import com.atlassian.performance.tools.aws.api.Aws
+import org.hamcrest.Matchers.isEmptyString
+import org.hamcrest.Matchers.not
+import org.junit.Assert.assertThat
+import org.junit.Test
+
+class AmiNameResolverIT {
+    @Test
+    fun worksInUsEast1() {
+        checkRegion(US_EAST_1)
+    }
+
+    @Test
+    fun worksInUsEast2() {
+        checkRegion(US_EAST_2)
+    }
+
+    @Test
+    fun worksInUsWest1() {
+        checkRegion(US_WEST_1)
+    }
+
+    @Test
+    fun worksInUsWest2() {
+        checkRegion(US_WEST_2)
+    }
+
+    @Test
+    fun worksInEuWest1() {
+        checkRegion(EU_WEST_1)
+    }
+
+    @Test
+    fun worksInEuWest2() {
+        checkRegion(EU_WEST_2)
+    }
+
+    @Test
+    fun worksInEuWest3() {
+        checkRegion(EU_WEST_3)
+    }
+
+    @Test
+    fun worksInEuCentral1() {
+        checkRegion(EU_CENTRAL_1)
+    }
+
+    @Test
+    fun worksInEuNorth1() {
+        checkRegion(EU_NORTH_1)
+    }
+
+    @Test
+    fun worksInApSouth1() {
+        checkRegion(AP_SOUTH_1)
+    }
+
+    @Test
+    fun worksInApSouthEast1() {
+        checkRegion(AP_SOUTHEAST_1)
+    }
+
+    @Test
+    fun worksInApSouthEast2() {
+        checkRegion(AP_SOUTHEAST_2)
+    }
+
+    @Test
+    fun worksInApNorthEast1() {
+        checkRegion(AP_SOUTHEAST_1)
+    }
+
+    @Test
+    fun worksInApNorthEast2() {
+        checkRegion(AP_SOUTHEAST_2)
+    }
+
+    private fun checkRegion(region: Regions) {
+        val vuAmi = AmiNameResolver.vuAmi(
```

Comment:
This logic (and test) belongs to [aws-resources](https://bitbucket.org/atlassian/aws-resources).

Reply:
I was thinking about it during the weekend and I'd like to skip the correction of this for now. This is mostly because I'm currently not sure how the aws-resources API should look like to properly address aws-infrastructure needs around AMIs.

I will raise an issue to move the AMI search responsibility to the proper place, but for now I need to bring the tests to the operational state without significant delay.

This PR we are commenting in should still improve the state and not make it worse.

Reply:
> I'm currently not sure how the aws-resources API should look like to properly address aws-infrastructure needs around AMIs.

This can stay as is for now, so no "significant delay".

Reply:
Ah, I thought that the `AmiNameResolver` is in `aws-resources`, but it's here. Sorry for the confusion.